frontend/packages/container-security/src/plugin.ts

```diff
@@ -67,6 +67,19 @@ const plugin: Plugin<ConsumedExtensions> = [
         ).then((m) => m.ImageManifestVulnDetailsPage),
     },
   },
+  {
+    type: 'Page/Route',
+    properties: {
+      exact: false,
+      path: `/k8s/ns/:ns/${referenceForModel(
+        ClusterServiceVersionModel,
+      )}/:appName/${referenceForModel(ImageManifestVulnModel)}/~/new`,
```

Comment:
I believe this should be:
```suggestion
      )}/:appName/${referenceForModel(ImageManifestVulnModel)}/~new`,
```
But I still don't think this solves the problem. We would still get a 404 on this page because now we are going to the details page without a name parameter at all.